plugins/woocommerce/src/Internal/Admin/CustomerEffortScoreTracks.php

```diff
@@ -387,10 +413,18 @@ private function enqueue_ces_survey_for_edited_shop_order() {
 		$this->enqueue_to_ces_tracks(
 			array(
 				'action'         => self::SHOP_ORDER_UPDATE_ACTION_NAME,
```

Comment:
This event is also triggered when the order is created, is that expected?

Reply:
I noticed this as well, but this was the preexisting behavior so I'd assumed it was expected. We could loop in @pmcpinto . I know it's triggered when we create the order manually through wc-admin. Glancing at the code it's triggered by a change in the `order` post status.

Reply:
the way it's working now is fine. Thanks for the heads up.